server.py

```diff
@@ -44,6 +56,16 @@ def download(path):
 def server_static(filepath):
     return static_file(filepath, root=os.path.join(dir_path, 'static'))
 
+@app.get('/ranking')
+def rankings():
+	people=[]
+	for u_name in usernames.keys():
```

Comment:
```suggestion
	for u_name in usernames:
```

When you do this it's a dictionary lookup which is constant time. The way you did it makes it a linear time lookup, thus losing all the benefit of being a dictionary.